### ISSUE TITLE
comparator.ne in selectFilter

### ISSUE THREAD

**Issue by vwen01** [react-bootstrap-table/react-bootstrap-table2#690]
![image](https://user-images.githubusercontent.com/19830762/49267579-18192780-f4c0-11e8-869d-bcda3c5ef631.png)

it seems like the Comparator.ne is not working in the above case, it keeps defaulting back to Comparator.EQ in my case?

**Comment by AllenFang** [comment]
@vwen01 yes, currently we only support EQ and LIKE, I will add NE ASAP, thanks 👍
